docs/modules/ROOT/pages/type-definitions/relationships.adoc

```diff
@@ -240,21 +260,3 @@ query {
     }
 }
 ----
-
-== Cardinality
-
-The Neo4j GraphQL Library has type definition requirements for "many" relationships, so given this example: 
-
-[source, graphql, indent=0]
-----
-type User {
-    name: String!
-    posts: [Post!]! @relationship(type: "HAS_POST", direction: OUT)
-}
-
-type Post {
-    name: String!
-}
-----
-
-The relationship at `User.posts` is considered a "many" relationship. Relationships such as the one above should always be of type `NonNullListType` and `NonNullNamedType`, meaning both the array and the type inside of it should have a `!`.
```

Comment:
**suggestion (blocking):** Any reason this has been removed?

Reply:
Nope, it is probably a conflict that I wrongly fixed